src/vs/workbench/contrib/notebook/browser/notebook.contribution.ts

```diff
@@ -757,161 +754,160 @@ configurationRegistry.registerConfiguration({
-		// },
-		// [DragAndDropEnabled]: {
-		// 	description: nls.localize('notebook.dragAndDrop.description', "Control whether the notebook editor should allow moving cells through drag and drop."),
-		// 	type: 'boolean',
-		// 	default: true,
-		// 	tags: ['notebookLayout']
-		// },
-		// [ConsolidatedRunButton]: {
-		// 	description: nls.localize('notebook.consolidatedRunButton.description', "Control whether extra actions are shown in a dropdown next to the run button."),
-		// 	type: 'boolean',
-		// 	default: false,
-		// 	tags: ['notebookLayout']
-		// },
-		// [GlobalToolbarShowLabel]: {
-		// 	description: nls.localize('notebook.globalToolbarShowLabel', "Control whether the actions on the notebook toolbar should render label or not."),
-		// 	type: 'boolean',
-		// 	enum: ['always', 'never', 'dynamic'],
-		// 	default: true,
-		// 	tags: ['notebookLayout']
-		// },
-		// [TextOutputLineLimit]: {
-		// 	description: nls.localize('notebook.textOutputLineLimit', "Control how many lines of text in a text output is rendered."),
-		// 	type: 'number',
-		// 	default: 30,
-		// 	tags: ['notebookLayout']
-		// },
-		// [NotebookSetting.markupFontSize]: {
-		// 	markdownDescription: nls.localize('notebook.markup.fontSize', "Controls the font size of rendered markup in notebooks. When set to `0`, 120% of `#editor.fontSize#` is used."),
-		// 	type: 'number',
-		// 	default: 0,
-		// 	tags: ['notebookLayout']
-		// },
-		// [NotebookSetting.cellEditorOptionsCustomizations]: editorOptionsCustomizationSchema
-		// [NotebookSetting.interactiveWindowCollapseCodeCells]: {
-		// 	markdownDescription: nls.localize('notebook.interactiveWindow.collapseCodeCells', "Controls whether code cells in the interactive window are collapsed by default."),
-		// 	type: 'string',
-		// 	enum: ['always', 'never', 'fromEditor'],
-		// 	default: 'fromEditor'
-		// },
-		// [NotebookSetting.outputLineHeight]: {
-		// 	markdownDescription: nls.localize('notebook.outputLineHeight', "Line height of the output text for notebook cells.\n - Values between 0 and 8 will be used as a multiplier with the font size.\n - Values greater than or equal to 8 will be used as effective values."),
-		// 	type: 'number',
-		// 	default: 22,
-		// 	tags: ['notebookLayout']
-		// },
-		// [NotebookSetting.outputFontSize]: {
-		// 	markdownDescription: nls.localize('notebook.outputFontSize', "Font size for the output text for notebook cells. When set to 0 `#editor.fontSize#` is used."),
-		// 	type: 'number',
-		// 	default: 0,
-		// 	tags: ['notebookLayout']
-		// },
-		// [NotebookSetting.outputFontFamily]: {
-		// 	markdownDescription: nls.localize('notebook.outputFontFamily', "The font family for the output text for notebook cells. When set to empty, the `#editor.fontFamily#` is used."),
-		// 	type: 'string',
-		// 	tags: ['notebookLayout']
-		// },
+		[NotebookSetting.displayOrder]: {
+			description: nls.localize('notebook.displayOrder.description', "Priority list for output mime types"),
```

Comment:
Is it clear these will only impact VS Code Notebooks? Might want to add a blurb on about that. (make sure to add updated strings to `src\sql\base\common\locConstants.ts`)

Reply:
Why do the localized strings need to be added into the sql folder's localized constants?

Reply:
If we change the ones that VS Code has we need to use our own because otherwise the localized versions of the strings get overwritten when we create our langpacks since it pulls in the ones used by VS Code. It's just a quirk with how we handle merging the localized strings currently (VS Code does theirs in a completely separate way so we pull those in manually IIRC)